### DISCUSSION TITLE
QskScrollArea view jitter problem

### DISCUSSION THREAD

**Discussion by hestrro** [uwerat/qskinny#493] [Q&A]
https://github.com/user-attachments/assets/2ba18778-d282-43f6-9284-8d4f6459b08f

Is the shaking in the video due to the animation effect? ​​How can I turn it off?

**Comment by uwerat** [comment]
I do not know enough about your use case, but it looks like you are displaying some sort of vector graphics, where positions get rounded to the closest integer in the target device coordinate system.

**Reply by hestrro** [reply]
This shaking animation will also occur after swiping in thumbnails
